modules/docs/mdx/12.0-UPGRADE-GUIDE.mdx

```diff
@@ -99,16 +108,6 @@ The following components are affected:
 > with `cs` in our
 > [documentation](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--cs-prop).
 
-### Popup, Dialog and Modal
-
-**PR:** [#2795](https://github.com/Workday/canvas-kit/pull/2795)
-
-### Toast
-
-**PR:** [#2795](https://github.com/Workday/canvas-kit/pull/2795)
-
-## Style Utility Updates
-
```

Comment:
I'm assuming this will just be a pull from `major` to add this back in?

Reply:
in v11, we didn't separate them like this, just created a list https://workday.github.io/canvas-kit/?path=/story/guides-upgrade-guides-v11-0--page#styling-api-and-css-tokens

Reply:
It's added here: https://github.com/Workday/canvas-kit/pull/2825/files#diff-3bd30c88399b3308c35d33a211aaf105e55425d97fe54399a2ca24d8e35d5e24R91